src/neuroconv/tools/aws/_submit_aws_batch_job.py

```diff
@@ -131,6 +141,29 @@ def submit_aws_batch_job(
         job_queue_name=job_queue_name, compute_environment_name=compute_environment_name, batch_client=batch_client
     )
 
+    # Create or fetch EFS volume and attach it to the job
+    efs_id = None
```

Comment:
Will similarly put this in a private helper later, to match style of other steps